### PR TITLE
Display forgotpw page

### PR DESCRIPTION
**Pull request by LouiaR**
Display signing signup & forgot page on the same route
Add custom css
Import signup and forgot password components inside login component 
See Trello for picture

### CODE REVIEW

File: frontend/src/Components/LandingPage/index.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import LoginForm from '../LoginForm';
 // import ForgotPassword from '../Users/forgot-password';  waiting Mohammad change to define route for this component
 import { deleteFlashMessage } from '../../actions/flashMessages';
 import './landing-page.css';
-import SignUpForm from '../Users/signUp';
+// import SignUpForm from '../Users/signUp';
```
Comment (DwinaTech) [inline frontend/src/Components/LandingPage/index.js]: if you do not need this code remove it, please.

File: frontend/src/Components/LandingPage/index.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ class LandingPage extends Component {
 
   showSignUpForm = () => {
     if (this.state.open) {
-      return <SignUpForm />
+      // return <SignUpForm />
```
Comment (DwinaTech) [inline frontend/src/Components/LandingPage/index.js]: ditto

File: frontend/src/Components/LandingPage/index.js (hunk 1 above)
Comment (LouiaR) [reply frontend/src/Components/LandingPage/index.js]: I did it

File: frontend/src/Components/LandingPage/index.js (hunk 2 above)
Comment (LouiaR) [reply frontend/src/Components/LandingPage/index.js]: I did it

**Pull request merged**